platform-api/tests/test_content.py

```
@@ -0,0 +1,95 @@
+import pytest
+from fastapi.testclient import TestClient
+from app import app
+from dependencies import get_sanity, get_settings
+from models.settings import WorkerSettings
+from unittest.mock import MagicMock
+
+client = TestClient(app)
+
+# 1. Create a Fake Sanity Client
+class MockSanityClient:
+    def __init__(self):
```

Ruff (0.15.6)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)